server/services/forms/schemas/bulk-document-add.js

```diff
@@ -1,13 +1,17 @@
 const { DOCUMENT_BULK_LIMIT } = require('../../../config').forContext('server');
 const Form = require('../form-builder');
 const { Component } = require('../component-builder');
+const { MIN_ALLOWABLE_YEAR } = require('../../../libs/dateHelpers');
 
 module.exports = () => Form()
     .withTitle('Create Bulk Cases')
     .withField(
         Component('date', 'DateReceived')
             .withValidator('required', 'Date received is required')
             .withValidator('isValidDate', 'Date received must be a valid date')
+            .withValidator('isValidDay', 'Date received must contain a real day')
+            .withValidator('isValidMonth', 'Date received must contain a real month')
+            .withValidator('isYearWithinRange', 'Date received must be after ' + MIN_ALLOWABLE_YEAR)
```

Comment:
Consider using template literals here (backtick strings)

Reply:
Updated here and in single document add